From e5a31558afca7d8b047b79c0fa57011f30c02881 Mon Sep 17 00:00:00 2001
From: Sheng Nan <b38800@freescale.com>
Date: Fri, 7 Dec 2012 16:54:36 +0800
Subject: [PATCH 374/442] ENGR00235665: mxc_v4l2_test: add YV12 capture format
 support in unit test

Kernel commit 98a5129e246eb69f4122b962a52017ba9c376925 has been supported
YV12 format in camera driver.

Add YV12 support in mxc_v4l2_capture unit test.

Signed-off-by: Sheng Nan <b38800@freescale.com>
---
 test/mxc_v4l2_test/mxc_v4l2_capture.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/test/mxc_v4l2_test/mxc_v4l2_capture.c b/test/mxc_v4l2_test/mxc_v4l2_capture.c
index 08197d4..81107c9 100644
--- a/test/mxc_v4l2_test/mxc_v4l2_capture.c
+++ b/test/mxc_v4l2_test/mxc_v4l2_capture.c
@@ -406,6 +406,7 @@ int process_cmdline(int argc, char **argv)
 			     (g_cap_fmt != V4L2_PIX_FMT_YUV422P) &&
 			     (g_cap_fmt != V4L2_PIX_FMT_UYVY) &&
 			     (g_cap_fmt != V4L2_PIX_FMT_YUYV) &&
+			     (g_cap_fmt != V4L2_PIX_FMT_YVU420) &&
                              (g_cap_fmt != V4L2_PIX_FMT_YUV420) )
                         {
                                 return -1;
-- 
1.8.0

